react-with-graphql/src/Components/AddTodo/AddTodo.js

```diff
@@ -6,10 +6,10 @@ import './AddTodo.css';
 
 let input;
 
-const updateCache = (cache, { data: { createtodo } }) => {
+const updateCache = (cache, { data: { createTodo } }) => {
```

Comment:
why are we not using a hook for these operations?

Reply:
Working on it